apps/passport-client/src/zapp/ZappServer.ts

```diff
@@ -270,6 +270,43 @@ class ZupassPODRPC extends BaseZappServer implements ParcnetPODRPC {
       this.getAdvice().showClient();
     });
   }
+
+  public async signPrefixed(entries: PODEntries): Promise<PODData> {
+    const origin = this.getContext().getState().zappOrigin;
+    if (
+      appConfig.zappRestrictOrigins &&
+      (!origin || !appConfig.zappAllowedSignerOrigins.includes(origin))
+    ) {
+      throw new Error("Origin not allowed to sign PODs");
+    }
+    if (!this.getPermissions().SIGN_POD) {
+      throw new MissingPermissionError("SIGN_POD", "pod.sign");
+    }
+
+    for (const name of Object.keys(entries)) {
+      if (!name.startsWith("_UNSAFE_")) {
```

Comment:
Why the leading underscore?  It's valid, just breaks the practice so far of reserved names like pod_ and zupass_
However this settles, we should document it as one of our "reserved" prefixes, meaning we don't keep people from using it, but recommend against using it for anything other than its intended purpose.
Or maybe we should reserve leading underscores in general?